pyproject.toml

```diff
@@ -18,6 +18,7 @@ select = [
   "E",
   "F",
   "C4",
+  "I",
```

Comment:
If this is too strict and we want to disable import sorting, we can remove this line.